otj-metrics-actuate/src/main/java/com/opentable/metrics/actuate/health/HealthIndicatorCodahaleConfiguration.java

```diff
@@ -0,0 +1,64 @@
+package com.opentable.metrics.actuate.health;
+
+import java.util.Map;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+import javax.inject.Inject;
+
+import com.codahale.metrics.health.HealthCheck;
+import com.codahale.metrics.health.HealthCheckRegistry;
+
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.boot.actuate.health.Status;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Registers {@link HealthIndicator} beans in the {@link HealthCheckRegistry}.
+ *
+ */
+@Configuration
+public class HealthIndicatorCodahaleConfiguration {
+
+    final HealthCheckRegistry registry;
+    final Map<String, HealthIndicator> checks;
+
+    @Inject
+    public HealthIndicatorCodahaleConfiguration(HealthCheckRegistry registry,   Map<String, HealthIndicator> checks) {
+        this.registry = registry;
+        this.checks = checks;
+    }
+
+    @PostConstruct
+    void postConstruct() {
+        checks.forEach((k, v) -> {
```

Comment:
You can use for descriptive variable names here, like checkName and healthIndicator